build_runner/lib/src/builder/post_process_builder.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2018, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:build/build.dart';
+
+/// A [PostProcessBuilder] runs in a special phase at the end of the build.
```

Comment:
[nit] noun-phrase.

"A builder which runes in a special phase at the end of the build."

Reply:
done